pkg/api/alertmanager.go

```diff
@@ -93,7 +93,7 @@ import (
 // swagger:parameters RouteCreateSilence
 type CreateSilenceParams struct {
 	// in:body
```

Comment:
Upstream uses the same nomenclature but unsure whenever we've deliberately changed it or this was unintended.

```
/ swagger:parameters postSilences
type PostSilencesParams struct {

	// HTTP Request Object
	HTTPRequest *http.Request `json:"-"`

	/*The silence to create
	  Required: true
	  In: body
	*/
	Silence *models.PostableSilence
}
```

Reply:
I don't think it was deliberately changed. @domasx2 can you confirm?